### ISSUE TITLE
Nextcloud crashes

### ISSUE THREAD

**Issue by jmue** [nextcloud/android#4168]
### Actual behaviour
- Nextcloud crashes while selecting app drawer entries

### Expected behaviour
- Nextcloud should not crash
 
### Steps to reproduce
1. Open nextcloud app
2. Open app drawer, click on 'Activities'
3.  Open app drawer, click on 'Favorites'

### Environment data
Android version: 9.0, 7.0

Device model: 
Emulator, Moto G

Stock or customized system:

Nextcloud app version:
3.7 RC3

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
    --------- beginning of crash
2019-06-21 22:30:30.917 25627-25627/com.nextcloud.client E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.nextcloud.client, PID: 25627
    org.parceler.ParcelerRuntimeException: Unable to find generated Parcelable class for com.owncloud.android.ui.events.SearchEvent, verify that your class is configured properly and that the Parcelable class com.owncloud.android.ui.events.SearchEvent$$Parcelable is generated by Parceler.
        at org.parceler.Parcels$ParcelCodeRepository.get(Parcels.java:154)
        at org.parceler.Parcels.wrap(Parcels.java:73)
        at org.parceler.Parcels.wrap(Parcels.java:57)
        at com.owncloud.android.ui.activity.DrawerActivity.handleSearchEvents(DrawerActivity.java:536)
        at com.owncloud.android.ui.activity.DrawerActivity.selectNavigationItem(DrawerActivity.java:450)
        at com.owncloud.android.ui.activity.DrawerActivity.access$300(DrawerActivity.java:119)
        at com.owncloud.android.ui.activity.DrawerActivity$3$1.run(DrawerActivity.java:344)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by jmue** [comment]
Btw. Why does 3.7 RC3 arrive at f-droid stable channel?

**Comment by ezaquarii** [comment]
Funny thing,but I see only 3.7 alpha1. I'm not very familiar with F-Droid, but does it mean it is a staged rollout or something?

**Comment by Babakz78** [comment]
I have the same problem

**Comment by new-on-github** [comment]
Same here. It even would't start. I just updated through fdroid and cannot go back to 3.6.2 through fdroid

**Comment by rotational467** [comment]
Same here, F-Droid offered 3.7.0 RC3 from whatever the current production release is, and won't allow a downgrade back past 3.7.0 RC1.  In my case (Pixel 3 XL stock) background sync works normally but the interactive client crashes when attempting to swipe through the file list in the **root folder only**.

Steps to reproduce:
1. Open Nextcloud 3.7.0 RC3 client.  Client loads and syncs with server properly.
2. Attempt to swipe down the file list in your root folder.  Client crashes.
3. Restart Nextcloud and open a subfolder without swiping the file list.
4. You can swipe up and down through the subfolder files without a crash.

There were no new objects created in my root storage folder since well before the problem started.

**Comment by AndyScherzinger** [comment]
fixed via  #4181
